python-mini-projects/projects/Create_a_simple_stopwatch/stopwatch.py

```diff
@@ -0,0 +1,81 @@
+import tkinter as Tkinter
+from datetime import datetime
+counter = 0
+running = False
+
+
+def counter_label(label):
+    def count():
+        if running:
+            global counter
+			# To manage the intial delay. 
+            if counter == 0:
+                display = 'Ready!'
+            else:
+                tt = datetime.utcfromtimestamp(counter)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by using `now` or `fromtimestamp` methods and passing `tzinfo` to them.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)